pkgs/development/python-modules/wandb/default.nix

```diff
@@ -274,6 +274,10 @@ buildPythonPackage rec {
     writableTmpDirAsHomeHook
   ];
 
+  preCheck = ''
+    export HOME=$(mktemp -d)
+  '';
+
```

Comment:
This should already be handled by `writableTmpDirAsHomeHook` (a few lines above).

Reply:
So, here's a fun question: why would a test fail when sandboxed under `writableTmpDirAsHomeHook` but not when you also set `HOME` to a temp dir? This is happening for `test_makedirs_raises_oserror__uses_temp_dir` during `nixpkgs-review`.

I disabled the failing test just now.

Reply:
Final change on this: moved the wandb change into #413654